megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -303,6 +301,27 @@ public void mouseDragged(MouseEvent e) {
     
     private static final Dimension LABEL_SPACER = new Dimension(5,0);
     private static final Dimension DEPENDENT_INSET = new Dimension(25,0);
+    
+    // Save some values to restore them when the dialog is canceled
+    private boolean savedFovHighlight;
+    private boolean savedFovDarken;
+    private boolean savedFovGrayscale;
+    private boolean savedAOHexShadows;
+    private boolean savedShadowMap;
+    private boolean savedHexInclines;
+    private boolean savedLevelhighlight;
+    private boolean savedFloatingIso;
+    private boolean savedMmSymbol;
+    private boolean savedTeamColoring;
+    private boolean savedUnitLabelBorder;
+    private boolean savedShowDamageDecal;
+    private boolean savedShowDamageLabel;
+    private String savedFovHighlightRingsRadii;
+    private String savedFovHighlightRingsColors;
+    private int savedFovHighlightAlpha;
+    private int savedFovDarkenAlpha;
+    private int savedNumStripesSlider;
+    HashMap<String, String> savedAdvancedOpt = new HashMap<>();
```

Comment:
Before I continue I have to get clarification on this: Why are we saving values for a revert and not just having the update only occur when the dialog is saved?

Reply:
Because these settings immediately show on the board or minimap and to do this they must be saved to the GUIPrefs.

Reply:
That makes sense. I'll finish my review post-D&D